ABaseEntity/ABaseEntity.cpp

```diff
@@ -4,7 +4,6 @@
 #include "System/NLogger.h"
 
 ABaseEntity::ABaseEntity() : IBaseEntity() {
-    Tags.Add(TAG_BASEENTITY);

```

Comment:
Remove redundant tags - these are to check whether the entity is a BaseEntity, but if we need this information, we can simply use a cast. It might be slightly more inefficient, but the situations in which we need to verify an Actor is a BaseEntity is very minimal.